src/XMLimport.cpp

```diff
@@ -800,6 +800,7 @@ void XMLimport::readHostPackage(Host* pHost)
     pHost->mFORCE_GA_OFF = (attributes().value("mFORCE_GA_OFF") == "yes");
     pHost->mFORCE_SAVE_ON_EXIT = (attributes().value("mFORCE_SAVE_ON_EXIT") == "yes");
     pHost->mEnableGMCP = (attributes().value("mEnableGMCP") == "yes");
+    pHost->mEnableMSSP = (attributes().value("mEnableMSSP") == "yes");
```

Comment:
This works for new profiles, but existing profiles - where the `mEnableMSSP` key doesn't exist in the latest save file - will get MSSP turned off :O so check for the keys existence before setting it. There's a few examples of that in there.

Reply:
Done!